.gersemirc

```diff
@@ -0,0 +1,7 @@
+line_length: 80
+cache: true
+indent: 2
+quiet: true
+warn_about_unknown_commands: false
+workers: 8
+list_expansion: favour-inlining
```

Comment:
Is this necessary ?
Is it going to be necessary on each and every project of ours ? 
Couldn't we just use some default settings instead ?

Reply:
I tried to put the options that would create the least changes. I think some of these _might_ be the default values though

Reply:
Do we care about the number of changes ? 
Those are already 3k on a 15k codebase, so IMHO it could be 2 or 3 times that and we would not see the difference, as long as it is in a single commit documented in `.git-blame-ignore-revs`

Reply:
Okay, in that case, I remove some of the options in the gersemirc file.